Makefile

```diff
@@ -46,13 +46,13 @@ speedleUnitTests:
 	go test ${TEST_OPTS} github.com/teramoby/speedle-plus/pkg/store/file
 	go test ${TEST_OPTS} github.com/teramoby/speedle-plus/pkg/store/etcd
 	go test ${TEST_OPTS} github.com/teramoby/speedle-plus/pkg/store/mongodb
-	go test ${TEST_OPTS} github.com/teramoby/speedle-plus/cmd/spctl/pdl
+	go test ${TEST_OPTS} github.com/teramoby/speedle-plus/pkg/pdl
 	go test ${TEST_OPTS} github.com/teramoby/speedle-plus/pkg/suid
 	go test ${TEST_OPTS} github.com/teramoby/speedle-plus/pkg/assertion
-	go clean -testcache
-	STORE_TYPE=etcd go test ${TEST_OPTS} github.com/teramoby/speedle-plus/pkg/eval
-	go clean -testcache
-	STORE_TYPE=mongodb go test ${TEST_OPTS} github.com/teramoby/speedle-plus/pkg/eval
+	#go clean -testcache
```

Comment:
why etcd and mongodb tests are commented out?

Reply:
I forgot to remove such comments. And I have removed the comments on etcd and mongo test cases and pushed the code again.